lib/util/AuthConfig.js

```diff
@@ -191,20 +263,29 @@ class AuthConfig extends Config {
     }
 
     _checkConfig() {
-        if (!this._json.accessToken || !this._json.expiresAt) {
-            if (this._location === AuthConfig.LOCATION.ENV_VARS) {
-                const Auth = require('../Auth');
-                const loginKey = process.env[ENV_VAR_IMPT_LOGINKEY];
-                const options = new Options({
-                    [Options.LOGIN_KEY] : loginKey,
-                    [Options.USER] : process.env[ENV_VAR_IMPT_USER],
-                    [Options.PASSWORD] : process.env[ENV_VAR_IMPT_PASSWORD],
+        if (this._location === AuthConfig.LOCATION.ENV_VARS) {
+            const Auth = require('../Auth');
+            const loginKey = process.env[ENV_VAR_IMPT_LOGINKEY];
```

Comment:
Any reason why we need this variable declared and not to use this inline just like `process.env[ENV_VAR_IMPT_USER],` and `process.env[ENV_VAR_IMPT_PASSWORD]`?

Reply:
I'm not sure what you mean here. Anyways, this a piece of code we didn't write, we just moved it from it's original state

Reply:
The const `loginKey` is used on L270, but no where else.  Since the other variables passed into Options are `process.env[]` directly, it would make sense to me to be consistent here.

Reply:
Yeah I see what you mean now, They are using `loginKey` and passing it to another function (`setAuthInfo`) so they just saved it so it's used somewhere else :)